core/src/main/scala/kafka/server/ReplicaManager.scala

```diff
@@ -2201,9 +2201,7 @@ class ReplicaManager(val config: KafkaConfig,
   private def updateLeaderAndFollowerMetrics(newFollowerTopics: Set[String]): Unit = {
     val leaderTopicSet = leaderPartitionsIterator.map(_.topic).toSet
     newFollowerTopics.diff(leaderTopicSet).foreach(brokerTopicStats.removeOldLeaderMetrics)
```

Comment:
`removeOldLeaderMetrics` tries to remove `REPLICATION_BYTES_OUT_PER_SEC` and `REASSIGNMENT_BYTES_OUT_PER_SEC` which apply across all topics too. Could you please remove them?